nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -27,6 +27,10 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 <!-- To avoid merge conflicts, consider adding your item at an arbitrary place in the list instead. -->
 
+- [`lib.customisation.extendMkDerivation`](https://nixos.org/manual/nixpkgs/unstable#function-library-lib.customisation.extendMkDerivation) is introduced to help define a new build helper supporting [fixed-point argument](https://nixos.org/manual/nixpkgs/unstable#chap-build-helpers-finalAttrs) (`(finalAttrs: { ... })`) based on an existing one with such support, or add fixed-point argument suppport to existing build helpers with little expression modification.
+
+  See https://nixos.org/manual/nixpkgs/unstable#sec-build-helper-extendMkDerivation for detail.
```

Comment:
Can't we use local links here? Otherwise the stable docs always refers to unstable which is suboptimal.

Reply:
Ideally, `https://nixos.org/manual/nixpkgs/24.05#chap-build-helpers-finalAttrs` should redirects to `https://nixos.org/manual/nixpkgs/unstable#chap-build-helpers-finalAttrs` before the branch-off, and the manual URLs inside the release note should all be versioned. This way, we'll always get relevant links before and after a release.

Reply:
Just opened an issue in the nixos-homepage repository: NixOS/nixos-homepage#1179.